### ISSUE TITLE
More than 4 player split screen

### ISSUE THREAD

**Issue by Engoni** [WinterSquire/AlphaRing#70]
Is it possible to make this mod support more than 4 player split screen in some way? Would be really great since lots of people have huge TVs nowadays!

**Comment by austinarchibald** [comment]
+1 for this! Nucleus Co-op is much more intensive and slow to start up. This would be fantastic.

**Comment by Engoni** [comment]
I tried using this mod and two nucleus instances for 8 player split screen, seemed to work, but the problem is I cant use more than 1 controller per instance!

**Comment by austinarchibald** [comment]
Yes I imagine it would require changes to the Nucleus handler code. Hopefully someone on their discord can do it!
